test/publishing_api/special_route_publisher_test.rb

```diff
@@ -22,13 +23,12 @@
     }
   }
 
-  describe ".publish" do
-    it "publishes the special routes" do
-      Timecop.freeze(Time.now) do
-        publishing_api.expects(:put_content_item).with(
-          special_route[:base_path],
-          {
-            content_id: special_route[:content_id],
+  let(:content_id) { special_route[:content_id] }
+
+  let(:payload) {
```

Comment:
The indenting here is a bit odd now. It might be worth it to inline `payload` into the `publishes the special routes` test, as it's only used there.